src/main/java/seedu/address/model/person/Person.java

```diff
@@ -2,11 +2,10 @@
 
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Ok I guess this is fine since its java.util.*

Reply:
fix towards the end.